extension/sysinfo/SysInfo.js

```diff
@@ -120,6 +123,29 @@ async function getThreadInfo() {
   }
 }
 
+async function getUptimeInfo() {
+  try {
+    const dnscrypt = await exec("ps -eo etimes,pid,cmd | grep dnsc[r]ypt.toml | awk '{print $1}'", {encoding: 'utf8'});
+    const dnsmasq = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep dnsmasq)$\" | tail -1 | awk '{print $1}'", {encoding: 'utf8'});
+    const bitbridge6 = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep bitbridge6)$\" | awk '{print $1}'", {encoding: 'utf8'});
+    const bitbridge7 = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep bitbridge7)$\" | awk '{print $1}'", {encoding: 'utf8'});
+    const openvpn = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep openvpn)$\" | awk '{print $1}'", {encoding: 'utf8'});
+    const fireMain = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep -x FireMain)$\" | awk '{print $1}'", {encoding: 'utf8'});
+    const fireApi = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep -x FireApi)$\" | awk '{print $1}'", {encoding: 'utf8'});
+    const fireMon = await exec("ps -eo etimes,pid | egrep -w \"$(pgrep -x FireMon)$\" | awk '{print $1}'", {encoding: 'utf8'});
+    uptimeInfo.dnscrypt = Number(dnscrypt.stdout.replace("\n", ""));
```

Comment:
Check how much time it will take. I'm worry that it may take up too much resource, since this api call is executed per two minutes.